### DISCUSSION TITLE
SVE kernels not included in `DYNAMIC_ARCH=1`?

### DISCUSSION THREAD

**Discussion by giordano** [OpenMathLib/OpenBLAS#4325] [Q&A] [answered]
The [`README.md`](https://github.com/OpenMathLib/OpenBLAS/blob/cb2950709fcb38b55d439e3a27416be7f2474931/README.md#support-for-multiple-targets-in-a-single-library) says

> On **ARMV8**, it enables support for CortexA53, CortexA57, CortexA72, CortexA73, Falkor, ThunderX, ThunderX2T99, TSV110 as well as generic ARMV8 cpus.

It'd look like SVE kernels (for example for A64FX) are not included when using dynamic detection, I was wondering if this is accurate or outdated information in the README?  Thanks!

**Comment by martin-frbg** [comment, marked as answer]
That's a bit outdated - with 0.3.24 (and I think 0.3.23 already, would need to check) DYNAMIC_ARCH would include the Neoverse V1 and N2 targets (if there is compiler/header support for sve on the build system). With 0.3.25 it includes the ARMV8SVE catch-all for everything SVE-capable that is not Neoverse.
